Microsoft.Toolkit.Uwp/Helpers/HttpHelper/HttpHelper.cs

```diff
@@ -104,6 +109,45 @@ public HttpHelper(int poolSize, IHttpFilter httpFilter)
             }
         }
 
+        public async Task<HttpHelperResponse> GetRequestAsync(HttpHelperRequest request, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            await _semaphore.WaitAsync().ConfigureAwait(false);
+
+            HttpClient client = null;
+
+            try
+            {
+                var httpRequestMessage = request.ToHttpRequestMessage();
+
+                client = GetHttpClientInstance();
+                foreach (var header in request.Headers)
+                {
+                    if (!client.DefaultRequestHeaders.ContainsKey(header.Key))
```

Comment:
instead of checking for key and then adding it, you could just use indexer

client.DefaultHeaders[key] = value;